### PR TITLE
Add wheel perspective, diameterRatio to constructor

### PR DESCRIPTION
**Pull request by 1N50MN14**
- Added optional wheel `perspective` argument to constructor
  
  - Added optional wheel `diameterRatio` argument to constructor
  
 - Changed `borderRadius` constuctor argument type to `BorderRadiusGeometry` - I understand the reasoning behind behind it being double, however, this grants more flexibility. In my case, I have three selects where I wanted the first to the left to have left radiuses, center none and right with right border radiuses.
 
 - Updated tests, example, changelog and bumped pubspec version

### CODE REVIEW

File: lib/selector_wheel/selector_wheel.dart [hunk 1]
```diff
@@ -64,8 +64,8 @@ class SelectorWheel<T> extends StatefulWidget {
 
   /// Selector wheel's highlighted area border radius.
   ///
-  /// @default 8.0
-  final double highlightBorderRadius;
+  /// @default BorderRadius.circual(8.0)
```
Comment (AlexLomm) [inline lib/selector_wheel/selector_wheel.dart]: There's a typo here

File: example/pubspec.lock [hunk 2]
```diff
@@ -385,7 +385,7 @@ packages:
       path: ".."
       relative: true
     source: path
-    version: "1.0.3"
+    version: "1.0.4"
```
Comment (AlexLomm) [inline example/pubspec.lock]: I think it makes more sense to make the version "1.1.0" since this is a breaking change 🤔

**Review by AlexLomm — CHANGES_REQUESTED** [review]

**Comment by AlexLomm** [comment]
Thank you for the contribution @1N50MN14! The changes mostly look good, just need tiny tweaks. After they're made I'll approve

**Comment by 1N50MN14** [comment]
@AlexLomm ahhh thanks for spotting the typo! I also bumped pubspec version to `1.1.0`  and updated changelog to reflect the new version and highlighted the breaking change.

**Comment by AlexLomm** [comment]
Thanks for the updates, I also added some minor tweaks.

**Comment by 1N50MN14** [comment]
Thanks @AlexLomm (and thanks for the package)

**Comment by AlexLomm** [comment]
You're welcome! Thanks for the contribution 👍

**Pull request merged**